hello-strongbox-nuget-visual-studio/README.md

```diff
@@ -21,7 +21,7 @@ To manage packages, you'll need to configure NuGet to acccess your storages by p
 3. To add a source, select +, edit the name, enter the URL or path in the Source control, and select Update. The source now appears in the selector drop-down.
 4. The `nuget-public` group repository should be at the top of the list.
 
-![NuGet Package Resources](https://github.com/strongbox/strongbox/wiki/resources%2Fimages%2Fnuget%2FNuGet-Package-Resources.png "Add new NuGet Package Resources")
+![NuGet Package Resources](https://github.com/strongbox/strongbox/docs/assets/images/NuGet-Package-Resources.png "Add new NuGet Package Resources")
```

Comment:
I'm afraid these links are incorrect. They should be pointing to the published ones under https://strongbox.github.io/. Could you please correct these? Thanks! :)

Reply:
@carlspring so it should be like https://strongbox.github.io/assets/images/*.png ?

Reply:
That's correct:
[`./docs/assets/**`](https://github.com/strongbox/strongbox-docs/tree/master/docs/assets) == `https://strongbox.github.io/assets/**`